related-projects.md

```diff
@@ -0,0 +1,63 @@
+<!--
+SPDX-FileCopyrightText: Copyright 2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Known implementations and projects related to the PSA Certified APIs
+
+Updated Dec 2022.
+ 
+## Software implementations
+
+- The reference implementation for PSA Crypto API can be found inside Mbed TLS:
+  [https://mbed-tls.readthedocs.io/en/latest/](https://mbed-tls.readthedocs.io/en/latest/)
+
+- Other PSA Certified APIs are included in the TF-M project:
+  [https://www.trustedfirmware.org/projects/tf-m/](https://www.trustedfirmware.org/projects/tf-m/)
+
+- Trusted Services implements those services in a Trusted Execution Environment on Cortex A processors
```

Comment:
There is no trailing colon here.

Can we make all of the list items consistent: either using a trailing colon before the link, or ending with "." and introduce the link with "See"

Reply:
done